scripts/dts/gen_defines.py

```diff
@@ -105,6 +106,117 @@ def parse_args():
     return parser.parse_args()
 
 
+def augment_node(node):
+    # Augment an EDT node with these zephyr-specific attributes, which
+    # are used to generate macros from it:
+    #
+    # - z_primary_ident: a primary node identifier based on the node's
+    #   path, PATH_<str2ident(node.path>)
```

Comment:
nit: your formatting here is messed up between < ( > )

Reply:
Fixed